diff --git a/bundle.json b/bundle.json
index 04256ac..b17a052 100644
--- a/bundle.json
+++ b/bundle.json
@@ -34,6 +34,7 @@
                 "eventhandler",
                 "camera_framework",
                 "graphic_2d",
+                "graphic_surface",
                 "distributed_hardware_fwk",
                 "device_security_level",
                 "device_manager",
diff --git a/distributedcamera.gni b/distributedcamera.gni
index 29e824c..075c55c 100644
--- a/distributedcamera.gni
+++ b/distributedcamera.gni
@@ -22,6 +22,8 @@ camerastandard_path = "//foundation/multimedia/camera_framework"
 
 graphicstandard_path = "//foundation/graphic/graphic_2d"
 
+graphicsurface_path = "//foundation/graphic/graphic_surface"
+
 common_path = "${distributedcamera_path}/common"
 
 services_path = "${distributedcamera_path}/services"
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/encodeoninputbufferavailable_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/encodeoninputbufferavailable_fuzzer/BUILD.gn
index 8cb3784..61fabb9 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/encodeoninputbufferavailable_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/encodeoninputbufferavailable_fuzzer/BUILD.gn
@@ -26,7 +26,7 @@ ohos_fuzztest("EncodeOnInputBufferAvailableFuzzTest") {
   include_dirs = [
     "${fwk_common_path}/utils/include",
     "${fwk_utils_path}/include/eventbus",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${services_path}/cameraservice/base/include",
   ]
 
@@ -72,7 +72,7 @@ ohos_fuzztest("EncodeOnInputBufferAvailableFuzzTest") {
     "c_utils:utils",
     "drivers_interface_display:libdisplay_composer_proxy_1.0",
     "eventhandler:libeventhandler",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hitrace:hitrace_meter",
   ]
 }
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/encodeonoutputbufferavailable_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/encodeonoutputbufferavailable_fuzzer/BUILD.gn
index 8f545ed..3272175 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/encodeonoutputbufferavailable_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/encodeonoutputbufferavailable_fuzzer/BUILD.gn
@@ -26,7 +26,7 @@ ohos_fuzztest("EncodeOnOutputBufferAvailableFuzzTest") {
   include_dirs = [
     "${fwk_common_path}/utils/include",
     "${fwk_utils_path}/include/eventbus",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${services_path}/cameraservice/base/include",
   ]
 
@@ -72,7 +72,7 @@ ohos_fuzztest("EncodeOnOutputBufferAvailableFuzzTest") {
     "c_utils:utils",
     "drivers_interface_display:libdisplay_composer_proxy_1.0",
     "eventhandler:libeventhandler",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hitrace:hitrace_meter",
   ]
 }
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicechannelneg_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicechannelneg_fuzzer/BUILD.gn
index da7bdb5..4d862a8 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicechannelneg_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicechannelneg_fuzzer/BUILD.gn
@@ -29,7 +29,7 @@ ohos_fuzztest("SinkServiceChannelNegFuzzTest") {
     "include",
     "${common_path}/include/constants",
     "${common_path}/include/utils",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${innerkits_path}/native_cpp/camera_sink/include",
     "${innerkits_path}/native_cpp/camera_sink/include/callback",
     "${services_path}/cameraservice/base/include",
@@ -70,7 +70,7 @@ ohos_fuzztest("SinkServiceChannelNegFuzzTest") {
     "c_utils:utils",
     "device_manager:devicemanagersdk",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hilog:libhilog",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceclosechannel_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceclosechannel_fuzzer/BUILD.gn
index 398d760..a9485bb 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceclosechannel_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceclosechannel_fuzzer/BUILD.gn
@@ -29,7 +29,7 @@ ohos_fuzztest("SinkServiceCloseChannelFuzzTest") {
     "include",
     "${common_path}/include/constants",
     "${common_path}/include/utils",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${innerkits_path}/native_cpp/camera_sink/include",
     "${innerkits_path}/native_cpp/camera_sink/include/callback",
     "${services_path}/cameraservice/base/include",
@@ -70,7 +70,7 @@ ohos_fuzztest("SinkServiceCloseChannelFuzzTest") {
     "c_utils:utils",
     "device_manager:devicemanagersdk",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hilog:libhilog",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicegetcamerainfo_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicegetcamerainfo_fuzzer/BUILD.gn
index f9c5b8a..f1e09e4 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicegetcamerainfo_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicegetcamerainfo_fuzzer/BUILD.gn
@@ -29,7 +29,7 @@ ohos_fuzztest("SinkServiceGetCameraInfoFuzzTest") {
     "include",
     "${common_path}/include/constants",
     "${common_path}/include/utils",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${innerkits_path}/native_cpp/camera_sink/include",
     "${innerkits_path}/native_cpp/camera_sink/include/callback",
     "${services_path}/cameraservice/base/include",
@@ -70,7 +70,7 @@ ohos_fuzztest("SinkServiceGetCameraInfoFuzzTest") {
     "c_utils:utils",
     "device_manager:devicemanagersdk",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hilog:libhilog",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceinitsink_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceinitsink_fuzzer/BUILD.gn
index 1f0c9fe..af97f05 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceinitsink_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceinitsink_fuzzer/BUILD.gn
@@ -29,7 +29,7 @@ ohos_fuzztest("SinkServiceInitSinkFuzzTest") {
     "include",
     "${common_path}/include/constants",
     "${common_path}/include/utils",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${innerkits_path}/native_cpp/camera_sink/include",
     "${innerkits_path}/native_cpp/camera_sink/include/callback",
     "${services_path}/cameraservice/base/include",
@@ -71,7 +71,7 @@ ohos_fuzztest("SinkServiceInitSinkFuzzTest") {
     "c_utils:utils",
     "device_manager:devicemanagersdk",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hilog:libhilog",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceopenchannel_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceopenchannel_fuzzer/BUILD.gn
index 761def8..c964656 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceopenchannel_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceopenchannel_fuzzer/BUILD.gn
@@ -29,7 +29,7 @@ ohos_fuzztest("SinkServiceOpenChannelFuzzTest") {
     "include",
     "${common_path}/include/constants",
     "${common_path}/include/utils",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${innerkits_path}/native_cpp/camera_sink/include",
     "${innerkits_path}/native_cpp/camera_sink/include/callback",
     "${services_path}/cameraservice/base/include",
@@ -70,7 +70,7 @@ ohos_fuzztest("SinkServiceOpenChannelFuzzTest") {
     "c_utils:utils",
     "device_manager:devicemanagersdk",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hilog:libhilog",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicereleasesink_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicereleasesink_fuzzer/BUILD.gn
index bc60eb1..1dc0268 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicereleasesink_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicereleasesink_fuzzer/BUILD.gn
@@ -29,7 +29,7 @@ ohos_fuzztest("SinkServiceReleaseSinkFuzzTest") {
     "include",
     "${common_path}/include/constants",
     "${common_path}/include/utils",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${innerkits_path}/native_cpp/camera_sink/include",
     "${innerkits_path}/native_cpp/camera_sink/include/callback",
     "${services_path}/cameraservice/base/include",
@@ -70,7 +70,7 @@ ohos_fuzztest("SinkServiceReleaseSinkFuzzTest") {
     "c_utils:utils",
     "device_manager:devicemanagersdk",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hilog:libhilog",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicestopcapture_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicestopcapture_fuzzer/BUILD.gn
index dbb7d7e..8ec4bd3 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicestopcapture_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicestopcapture_fuzzer/BUILD.gn
@@ -29,7 +29,7 @@ ohos_fuzztest("SinkServiceStopCaptureFuzzTest") {
     "include",
     "${common_path}/include/constants",
     "${common_path}/include/utils",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${innerkits_path}/native_cpp/camera_sink/include",
     "${innerkits_path}/native_cpp/camera_sink/include/callback",
     "${services_path}/cameraservice/base/include",
@@ -70,7 +70,7 @@ ohos_fuzztest("SinkServiceStopCaptureFuzzTest") {
     "c_utils:utils",
     "device_manager:devicemanagersdk",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hilog:libhilog",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicesubscribelocalhardware_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicesubscribelocalhardware_fuzzer/BUILD.gn
index 8bd7444..040a7dc 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicesubscribelocalhardware_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkservicesubscribelocalhardware_fuzzer/BUILD.gn
@@ -29,7 +29,7 @@ ohos_fuzztest("SinkServiceSubscribeLocalHardwareFuzzTest") {
     "include",
     "${common_path}/include/constants",
     "${common_path}/include/utils",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${innerkits_path}/native_cpp/camera_sink/include",
     "${innerkits_path}/native_cpp/camera_sink/include/callback",
     "${services_path}/cameraservice/base/include",
@@ -70,7 +70,7 @@ ohos_fuzztest("SinkServiceSubscribeLocalHardwareFuzzTest") {
     "c_utils:utils",
     "device_manager:devicemanagersdk",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hilog:libhilog",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
diff --git a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceunsubscribelocalhardware_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceunsubscribelocalhardware_fuzzer/BUILD.gn
index 8196d45..4978228 100644
--- a/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceunsubscribelocalhardware_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sinkfuzztest/sinkserviceunsubscribelocalhardware_fuzzer/BUILD.gn
@@ -30,7 +30,7 @@ ohos_fuzztest("SinkServiceUnsubscribeLocalHardwareFuzzTest") {
     "include",
     "${common_path}/include/constants",
     "${common_path}/include/utils",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${innerkits_path}/native_cpp/camera_sink/include",
     "${innerkits_path}/native_cpp/camera_sink/include/callback",
     "${services_path}/cameraservice/base/include",
@@ -71,7 +71,7 @@ ohos_fuzztest("SinkServiceUnsubscribeLocalHardwareFuzzTest") {
     "c_utils:utils",
     "device_manager:devicemanagersdk",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hilog:libhilog",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
diff --git a/interfaces/inner_kits/native_cpp/test/sourcefuzztest/decodeoninputbufferavailable_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sourcefuzztest/decodeoninputbufferavailable_fuzzer/BUILD.gn
index 90cd852..4a5f518 100644
--- a/interfaces/inner_kits/native_cpp/test/sourcefuzztest/decodeoninputbufferavailable_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sourcefuzztest/decodeoninputbufferavailable_fuzzer/BUILD.gn
@@ -26,7 +26,7 @@ ohos_fuzztest("DecodeOnInputBufferAvailableFuzzTest") {
   include_dirs = [
     "${fwk_common_path}/utils/include",
     "${fwk_utils_path}/include/eventbus",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${services_path}/cameraservice/base/include",
   ]
 
@@ -71,7 +71,7 @@ ohos_fuzztest("DecodeOnInputBufferAvailableFuzzTest") {
     "av_codec:av_codec_client",
     "c_utils:utils",
     "eventhandler:libeventhandler",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hitrace:hitrace_meter",
   ]
 }
diff --git a/interfaces/inner_kits/native_cpp/test/sourcefuzztest/decodeonoutputbufferavailable_fuzzer/BUILD.gn b/interfaces/inner_kits/native_cpp/test/sourcefuzztest/decodeonoutputbufferavailable_fuzzer/BUILD.gn
index ac66083..c0d537b 100644
--- a/interfaces/inner_kits/native_cpp/test/sourcefuzztest/decodeonoutputbufferavailable_fuzzer/BUILD.gn
+++ b/interfaces/inner_kits/native_cpp/test/sourcefuzztest/decodeonoutputbufferavailable_fuzzer/BUILD.gn
@@ -26,7 +26,7 @@ ohos_fuzztest("DecodeOnOutputBufferAvailableFuzzTest") {
   include_dirs = [
     "${fwk_common_path}/utils/include",
     "${fwk_utils_path}/include/eventbus",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${services_path}/cameraservice/base/include",
   ]
 
@@ -71,7 +71,7 @@ ohos_fuzztest("DecodeOnOutputBufferAvailableFuzzTest") {
     "av_codec:av_codec_client",
     "c_utils:utils",
     "eventhandler:libeventhandler",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hitrace:hitrace_meter",
   ]
 }
diff --git a/services/cameraservice/cameraoperator/client/BUILD.gn b/services/cameraservice/cameraoperator/client/BUILD.gn
index a0a6641..798c547 100644
--- a/services/cameraservice/cameraoperator/client/BUILD.gn
+++ b/services/cameraservice/cameraoperator/client/BUILD.gn
@@ -28,7 +28,7 @@ ohos_shared_library("distributed_camera_client") {
   }
   stack_protector_ret = true
   include_dirs = [
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/input",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/output",
@@ -84,7 +84,7 @@ ohos_shared_library("distributed_camera_client") {
     "drivers_interface_camera:metadata",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
     "drivers_peripheral_display:hdi_gralloc_client",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
     "samgr:samgr_proxy",
diff --git a/services/cameraservice/cameraoperator/client/test/sample/BUILD.gn b/services/cameraservice/cameraoperator/client/test/sample/BUILD.gn
index 328575b..a293c06 100644
--- a/services/cameraservice/cameraoperator/client/test/sample/BUILD.gn
+++ b/services/cameraservice/cameraoperator/client/test/sample/BUILD.gn
@@ -20,7 +20,7 @@ config("module_private_config") {
   include_dirs = [
     "//third_party/cJSON",
     "//third_party/jsoncpp/include",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/input",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/output",
diff --git a/services/cameraservice/cameraoperator/client/test/unittest/common/cameraoperator/BUILD.gn b/services/cameraservice/cameraoperator/client/test/unittest/common/cameraoperator/BUILD.gn
index 69e5212..c7e5902 100644
--- a/services/cameraservice/cameraoperator/client/test/unittest/common/cameraoperator/BUILD.gn
+++ b/services/cameraservice/cameraoperator/client/test/unittest/common/cameraoperator/BUILD.gn
@@ -22,7 +22,7 @@ config("module_private_config") {
   include_dirs = [
     "//third_party/cJSON",
     "//third_party/jsoncpp/include",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/input",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/output",
diff --git a/services/cameraservice/cameraoperator/handler/BUILD.gn b/services/cameraservice/cameraoperator/handler/BUILD.gn
index dc2cd24..bbb974e 100644
--- a/services/cameraservice/cameraoperator/handler/BUILD.gn
+++ b/services/cameraservice/cameraoperator/handler/BUILD.gn
@@ -31,7 +31,7 @@ ohos_shared_library("distributed_camera_handler") {
     "//third_party/cJSON",
     "//third_party/jsoncpp/include",
     "//third_party/json/include",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/input",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/output",
@@ -78,7 +78,7 @@ ohos_shared_library("distributed_camera_handler") {
     "drivers_interface_camera:metadata",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
     "drivers_peripheral_display:hdi_gralloc_client",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "ipc:ipc_core",
   ]
 
diff --git a/services/cameraservice/cameraoperator/handler/test/unittest/common/dcamerahandler/BUILD.gn b/services/cameraservice/cameraoperator/handler/test/unittest/common/dcamerahandler/BUILD.gn
index ddcce16..5b86b21 100644
--- a/services/cameraservice/cameraoperator/handler/test/unittest/common/dcamerahandler/BUILD.gn
+++ b/services/cameraservice/cameraoperator/handler/test/unittest/common/dcamerahandler/BUILD.gn
@@ -20,7 +20,7 @@ module_out_path = "distributed_camera/dcamera_handler_test"
 config("module_private_config") {
   visibility = [ ":*" ]
   include_dirs = [
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/input",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/output",
diff --git a/services/cameraservice/sinkservice/BUILD.gn b/services/cameraservice/sinkservice/BUILD.gn
index 8068f13..e130cc1 100644
--- a/services/cameraservice/sinkservice/BUILD.gn
+++ b/services/cameraservice/sinkservice/BUILD.gn
@@ -29,7 +29,7 @@ ohos_shared_library("distributed_camera_sink") {
   stack_protector_ret = true
   include_dirs = [
     "${av_transport_path}/common/include",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/input",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/output",
@@ -133,7 +133,7 @@ ohos_shared_library("distributed_camera_sink") {
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
     "drivers_peripheral_display:hdi_gralloc_client",
     "eventhandler:libeventhandler",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "ipc:ipc_core",
     "safwk:system_ability_fwk",
     "samgr:samgr_proxy",
diff --git a/services/cameraservice/sinkservice/test/unittest/common/distributedcamera/BUILD.gn b/services/cameraservice/sinkservice/test/unittest/common/distributedcamera/BUILD.gn
index 52a57b1..c4fd53c 100644
--- a/services/cameraservice/sinkservice/test/unittest/common/distributedcamera/BUILD.gn
+++ b/services/cameraservice/sinkservice/test/unittest/common/distributedcamera/BUILD.gn
@@ -22,7 +22,7 @@ config("module_private_config") {
   include_dirs = [
     "//third_party/cJSON",
     "//third_party/jsoncpp/include",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/input",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/output",
diff --git a/services/cameraservice/sinkservice/test/unittest/common/distributedcameramgr/BUILD.gn b/services/cameraservice/sinkservice/test/unittest/common/distributedcameramgr/BUILD.gn
index 018c128..3a24128 100644
--- a/services/cameraservice/sinkservice/test/unittest/common/distributedcameramgr/BUILD.gn
+++ b/services/cameraservice/sinkservice/test/unittest/common/distributedcameramgr/BUILD.gn
@@ -20,7 +20,7 @@ module_out_path = "distributed_camera/dcamera_sink_mgr_test"
 config("module_private_config") {
   visibility = [ ":*" ]
   include_dirs = [
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/input",
     "${camerastandard_path}/interfaces/inner_api/native/camera/include/output",
diff --git a/services/cameraservice/sourceservice/BUILD.gn b/services/cameraservice/sourceservice/BUILD.gn
index e420464..df44361 100644
--- a/services/cameraservice/sourceservice/BUILD.gn
+++ b/services/cameraservice/sourceservice/BUILD.gn
@@ -59,7 +59,7 @@ ohos_shared_library("distributed_camera_source") {
     "${services_path}/data_process/include/interfaces",
     "${services_path}/data_process/include/pipeline",
     "${services_path}/data_process/include/utils",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${fwk_common_path}/utils/include",
     "${fwk_innerkits_path}/include",
   ]
@@ -129,7 +129,7 @@ ohos_shared_library("distributed_camera_source") {
     "c_utils:utils",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
     "eventhandler:libeventhandler",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hdf_core:libhdi",
     "hdf_core:libpub_utils",
     "hicollie:libhicollie",
diff --git a/services/cameraservice/sourceservice/test/unittest/common/distributedcamera/BUILD.gn b/services/cameraservice/sourceservice/test/unittest/common/distributedcamera/BUILD.gn
index a274a90..1f03389 100644
--- a/services/cameraservice/sourceservice/test/unittest/common/distributedcamera/BUILD.gn
+++ b/services/cameraservice/sourceservice/test/unittest/common/distributedcamera/BUILD.gn
@@ -47,7 +47,7 @@ config("module_private_config") {
     "//third_party/cJSON",
     "//third_party/jsoncpp/include",
     "//drivers/peripheral/base/",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
   ]
 }
 
diff --git a/services/cameraservice/sourceservice/test/unittest/common/distributedcameramgr/BUILD.gn b/services/cameraservice/sourceservice/test/unittest/common/distributedcameramgr/BUILD.gn
index 458650c..924b80f 100644
--- a/services/cameraservice/sourceservice/test/unittest/common/distributedcameramgr/BUILD.gn
+++ b/services/cameraservice/sourceservice/test/unittest/common/distributedcameramgr/BUILD.gn
@@ -48,7 +48,7 @@ config("module_private_config") {
     "//third_party/cJSON",
     "//third_party/jsoncpp/include",
     "//drivers/peripheral/base/",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${feeding_smoother_path}",
     "${feeding_smoother_path}/base",
     "${feeding_smoother_path}/derived",
diff --git a/services/channel/test/unittest/common/channel/BUILD.gn b/services/channel/test/unittest/common/channel/BUILD.gn
index b5d17d8..d8911f0 100644
--- a/services/channel/test/unittest/common/channel/BUILD.gn
+++ b/services/channel/test/unittest/common/channel/BUILD.gn
@@ -47,7 +47,7 @@ config("module_private_config") {
     "${services_path}/data_process/include/utils",
     "${innerkits_path}/native_cpp/camera_source/include",
     "${innerkits_path}/native_cpp/camera_source/include/callback",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "//third_party/cJSON",
     "//third_party/jsoncpp/include",
     "${feeding_smoother_path}/base",
diff --git a/services/data_process/BUILD.gn b/services/data_process/BUILD.gn
index 3cbe5ad..522adb3 100644
--- a/services/data_process/BUILD.gn
+++ b/services/data_process/BUILD.gn
@@ -30,7 +30,7 @@ ohos_shared_library("distributed_camera_data_process") {
   include_dirs = [
     "${fwk_common_path}/utils/include",
     "${fwk_utils_path}/include/eventbus",
-    "${graphicstandard_path}/frameworks/surface/include",
+    "${graphicsurface_path}/surface/include",
     "${services_path}/cameraservice/base/include",
   ]
 
@@ -106,7 +106,7 @@ ohos_shared_library("distributed_camera_data_process") {
     "c_utils:utils",
     "drivers_interface_display:libdisplay_composer_proxy_1.0",
     "eventhandler:libeventhandler",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hitrace:hitrace_meter",
     "safwk:system_ability_fwk",
     "samgr:samgr_proxy",
diff --git a/services/data_process/test/unittest/common/pipeline/BUILD.gn b/services/data_process/test/unittest/common/pipeline/BUILD.gn
index 12dde88..3c334c2 100644
--- a/services/data_process/test/unittest/common/pipeline/BUILD.gn
+++ b/services/data_process/test/unittest/common/pipeline/BUILD.gn
@@ -59,7 +59,7 @@ ohos_unittest("DCameraDataProcessPipelineTest") {
     "av_codec:av_codec_client",
     "c_utils:utils",
     "eventhandler:libeventhandler",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hitrace:hitrace_meter",
   ]
 
diff --git a/services/data_process/test/unittest/common/pipeline_node/BUILD.gn b/services/data_process/test/unittest/common/pipeline_node/BUILD.gn
index 26de7b6..21dcabd 100644
--- a/services/data_process/test/unittest/common/pipeline_node/BUILD.gn
+++ b/services/data_process/test/unittest/common/pipeline_node/BUILD.gn
@@ -72,7 +72,7 @@ ohos_unittest("DCameraDataProcessPipelineNodeTest") {
     "c_utils:utils",
     "drivers_interface_display:libdisplay_composer_proxy_1.0",
     "eventhandler:libeventhandler",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hitrace:hitrace_meter",
   ]
 
diff --git a/test/distributedcameraclienttest/BUILD.gn b/test/distributedcameraclienttest/BUILD.gn
index 608bba7..169c09c 100644
--- a/test/distributedcameraclienttest/BUILD.gn
+++ b/test/distributedcameraclienttest/BUILD.gn
@@ -66,7 +66,7 @@ ohos_moduletest_suite("SubDctsdisCameraClientTest") {
     "distributed_hardware_fwk:libdhfwk_sdk",
     "drivers_interface_camera:metadata",
     "drivers_interface_distributed_camera:libdistributed_camera_provider_proxy_1.0",
-    "graphic_2d:surface",
+    "graphic_surface:surface",
     "hilog:libhilog",
     "ipc:ipc_core",
   ]
